### PR TITLE
ci(github): tag PRs with merge conflicts automatically

### PR DESCRIPTION
**Pull request by MrTimscampi**

### CODE REVIEW

File: .github/workflows/merge-conflicts.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: "Merge Conflicts"
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    if: github.repository == 'jellyfin/jellyfin-web'
```
Comment (ferferga) [inline .github/workflows/merge-conflicts.yml:10]:
```suggestion
    if: github.repository == 'jellyfin/jellyfin-vue'
```

**Review by ferferga — CHANGES_REQUESTED** [review]
I think we also need to have the tag created beforehand for this to work

File: .github/workflows/merge-conflicts.yml (hunk 1 above)
Comment (MrTimscampi) [reply .github/workflows/merge-conflicts.yml:10]: Oh, heh, don't copy paste without looking :p

**Comment by ferferga** [comment]
We could maybe add ui tag as well when changes to components, pages or layouts happen?

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - "*/**/*.spec.js"
+
+github_actions:
+  - ".github/workflows/*"
+
+vuex:
+  - "store/**/*.ts"
+
+plugins:
+  - "plugins/**/.ts"
```
Comment (ThibaultNocchi) [inline .github/labeler.yml:11]:
```suggestion
  - "plugins/**/*.ts"
```

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+tests:
+  - "*/**/*.spec.js"
```
Comment (ThibaultNocchi) [inline .github/labeler.yml:2]:
```suggestion
  - "**/*.spec.js"
```

Couldn't that be just that?

File: .github/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - ".github/workflows/*"
+
+vuex:
+  - "store/**/*.ts"
+
+plugins:
+  - "plugins/**/.ts"
+
+vue:
+  - "*/**/*.vue"
```
Comment (ThibaultNocchi) [inline .github/labeler.yml:14]: Same here, maybe we can directly start with `**/.vue`

**Review by ThibaultNocchi — CHANGES_REQUESTED** [review]

**Review by ThibaultNocchi — APPROVED** [review]
LGTM

**Review by ferferga — APPROVED** [review]

**Pull request merged**